docs/02-app/02-api-reference/04-functions/userAgent.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: userAgent
+description: The userAgent helper extends the Web Request API with additional properties and methods to interact with the user agent object from the request.
+---
+
+{/* The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component. */}
+
+The `userAgent` helper extends the [Web Request API](https://developer.mozilla.org/docs/Web/API/Request) with additional properties and methods to interact with the user agent object from the request.
```

Comment:
Is this middleware only (I haven't come across it before)? If so, we should probably note it.

Reply:
Should be available everywhere you have the request, is my understanding.

Reply:
@leerob is it available in RSC? e.g. `app/page.tsx`

Reply:
No, but it is available in Route Handlers (where you have the request object).